addons/l10n_account_withholding/models/account_tax.py

```diff
@@ -0,0 +1,25 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    # ------------------
+    # Fields declaration
+    # ------------------
+
+    type_tax_use = fields.Selection(
+        selection_add=[
+            ('sales_wth', "Sales Withholding"),
+            ('purchases_wth', "Purchases Withholding"),
+        ],
+        ondelete={'sales_wth': 'set default', 'purchases_wth': 'set default'}
+    )
```

Comment:
Why do we need to change how the `type_tax_use` works?
In Italy we differentiate them with a `withholding_kind` to better understand if it's withholding for social security or for the simplified VAT case (of professionals who don't have VAT for example).
https://github.com/odoo/odoo/blob/18.0/addons/l10n_it_edi_withholding/models/account_tax.py#L89

I find it uneasy to have `sales_wth` taxes not being selected when you search for `sale` taxes. What do you think?

Reply:
Discussed with Benj.
This is based on what as done in AR where they require you to have a type tax use at none to then see a new field that you can use to select the withholding type.
This way is easier for the user, and in practice taking that as a tax type is not wrong.

If more specific types are needed for specific countries, this can be handled in a "specialized" field in that localization